packages/roosterjs-editor-dom/lib/table/applyTableFormat.ts

```diff
@@ -62,6 +63,7 @@ function setCellColor(cells: VCell[][], format: TableFormat) {
                 } else {
                     setColor(cell.td, TRANSPARENT, true /** isBackgroundColor*/);
                 }
+                adaptFontColorToBackgroundColor(cell.td);
```

Comment:
I saw this function is always used after setColor is called, prefer integrate it into setColor() and add an optional parameter to specify if we need this operation, then no need to export this function